packaging/suse/container/Dockerfile

```
@@ -11,10 +11,10 @@ WORKDIR /build/web/assets
 RUN npm run tailwind:build
 RUN npm run build
 
-FROM bci/bci-base:15.4 AS release
+FROM bci/bci-base:15.6 AS release
```

since we're at it, we should start using the full registry path
```suggestion
FROM registry.suse.com/bci/bci-base:15.6 AS release
```